elser/track.json

```diff
@@ -0,0 +1,31 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Benchmark weighted terms queries on ELSER tokens",
+  "indices": [
+    {
+      "name": "msmarco-passage-collection-elser",
+      "body": "msmarco-passage-collection-elser.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "msmarco-passage-collection-elser",
+      "documents": [
+        {
+          "target-index": "msmarco-passage-collection-elser",
+          "source-file": "msmarco-passage-collection-elser-documents.json.bz2",
+          "document-count": 100000,
+          "compressed-bytes": 78074178,
+          "uncompressed-bytes": 254501902
+        }
+      ]
```

Comment:
Can you please add support for `--test-mode`? https://esrally.readthedocs.io/en/stable/adding_tracks.html#adding-support-for-test-mode

We use this in the CI to make sure your track does not have errors.